### PR TITLE
[ZEPPELIN-4758] Additional cleanup for Java 8 support

### PR DESCRIPTION
**Pull request by alexott**
### What is this PR for?

Improve Java 8 support:

- Remove obsolete Java options
- add the check for minimum Java version (Java 8u151) to `zeppelin.sh` & `interpreter.sh`

### What type of PR is it?

Refactoring

### What is the Jira issue?
* ZEPPELIN-4758

### How should this be tested?
* https://travis-ci.org/github/alexott/zeppelin/builds/677566776

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -170,7 +170,7 @@ before_install:
   - gitlog=$(git log $TRAVIS_COMMIT_RANGE 2>/dev/null) || gitlog=""
   - clearcache=$(echo $gitlog | grep -c -E "clear bower|bower clear" || true)
   - if [ "$hasbowerchanged" -gt 0 ] || [ "$clearcache" -gt 0 ]; then echo "Clearing bower_components cache"; rm -r zeppelin-web/bower_components; npm cache verify; else echo "Using cached bower_components."; fi
-  - echo "MAVEN_OPTS='-Xms1024M -Xmx2048M -XX:MaxPermSize=1024m -XX:-UseGCOverheadLimit -Dorg.slf4j.simpleLogger.defaultLogLevel=warn'" >> ~/.mavenrc
+  - echo "MAVEN_OPTS='-Xms1024M -Xmx2048M -XX:-UseGCOverheadLimit -Dorg.slf4j.simpleLogger.defaultLogLevel=warn'" >> ~/.mavenrc
```
Comment (zjffdu) [inline .travis.yml:173]: Should we replace it with `-XX:MaxMetaspaceSize`

File: bin/common.sh [hunk 2]
```diff
@@ -62,6 +62,20 @@ fi
...
+function check_java_version() {
+    java_ver_output=$("${JAVA:-java}" -version 2>&1)
+    jvmver=$(echo "$java_ver_output" | grep '[openjdk|java] version' | awk -F'"' 'NR==1 {print $2}' | cut -d\- -f1)
+    JVM_VERSION=$(echo "$jvmver"|sed -e 's|^\([0-9]\+\)\..*$|\1|')
+    if [ "$JVM_VERSION" = "1" ]; then
+        JVM_VERSION=$(echo "$jvmver"|sed -e 's|^1\.\([0-9]\+\)\..*$|\1|')
+    fi
+
+    if [ "$JVM_VERSION" -lt 8 ] || ([ "$JVM_VERSION" -eq 8 ] && [ "${jvmver#*_}" -lt 151 ]) ; then
+        echo "Apache Zeppelin requires either Java 8 update 151 or newer"
```
Comment (zjffdu) [inline bin/common.sh:74]: Why need update 151 later ?

Comment (alexott) [reply bin/common.sh:74]: I selected 151 as it has unlimited security policy by default, and also had many security fixes (although 181+ is even better).  I can remove this specific requirement

File: .travis.yml (hunk 1 above)
Comment (alexott) [reply .travis.yml]: done. Although they aren't completely the same...

**Comment by zjffdu** [comment]
LGTM

**Pull request closed without merging**